release/src/main/scripts/build_release_candidate.sh

```diff
@@ -113,7 +113,7 @@ if [[ $confirmation = "y" ]]; then
   echo "-------------Staging Java Artifacts into Maven---------------"
```

Comment:
Add that `ssh-add` trick to cache ssh password?

Reply:
I wasn't sure the best way to do that. Maybe we can add it in a later PR.